x/observer/keeper/msg_server_update_operational_flags.go

```diff
@@ -0,0 +1,28 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	authoritytypes "github.com/zeta-chain/node/x/authority/types"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+func (k msgServer) UpdateOperationalFlags(
```

Comment:
Please add a comment to explain how these values would be used

Reply:
The values are already commented on the proto side?

Reply:
Yeah , but it might be usefull to have a comment here as well .